### PR TITLE
updated cutdown driver with cutdown format

### PR DESCRIPTION
**Pull request by IonLi09**
added cutdown sequence number to logger.

### CODE REVIEW

File: EosPayload/drivers/cutdown_driver.py
Comment (DarylDohner) [inline EosPayload/drivers/cutdown_driver.py:84]: don't throw, you should instead use logger.error()
also consider a more descriptive message like "incorrect type X, expected CutDown"

File: EosPayload/drivers/cutdown_driver.py
Comment (DarylDohner) [inline EosPayload/drivers/cutdown_driver.py:82]: so clearly this wasnt tested...you need Packet.decode here

File: EosPayload/drivers/cutdown_driver.py
Comment (DarylDohner) [inline EosPayload/drivers/cutdown_driver.py:7]: you need to update requirements.txt to the correct EosLib version

**Review by DarylDohner — DISMISSED** [review]
a few things
1. see comments
2. you need to actually test this, i would recommend either writing a quick script that sends a cutdown command over mqtt or just hooking up the radio / ground station, not sure how far along that is
3. there are a few places in the cutdown driver that call time.sleep, please replace these with self.thread_sleep

File: EosPayload/drivers/cutdown_driver.py
Comment (VanL15) [inline EosPayload/drivers/cutdown_driver.py:87]: Make the 'i' in 'incorrect' capital

File: EosPayload/drivers/cutdown_driver.py
Comment (VanL15) [inline EosPayload/drivers/cutdown_driver.py:105]: make the 'r' in 'received' capital

File: EosPayload/drivers/radio_driver.py
Comment (VanL15) [inline EosPayload/drivers/radio_driver.py:98]: Remove this comment

**Review by VanL15 — CHANGES_REQUESTED** [review]
see comments, wait until ping is merged

**Review by aryanbattula — DISMISSED** [review]
good job!

**Review by aryanbattula — APPROVED** [review]
lgtm

**Pull request merged**
